pkg/commands/process/settings/rules/javascript/lang/dangerous_insert_html.yml

```
@@ -41,8 +41,8 @@ auxiliary:
           - variable: STRING_LITERAL
             detection: string_literal
             contains: false
-      - sanitize($<_>)
-      - sanitizeHTML($<_>)
+      - $<_>.sanitize($<...>$<_>$<...>)
+      - $<_>.sanitizeHTML($<...>$<_>$<...>)
```

```suggestion
      - $<_>.sanitize()
      - $<_>.sanitizeHTML()
```

Why not simply this?

---

think it will match only empty function calls?

---

will test it

---

seems to work